blog/2012/02/why-we-created-julia.md

```diff
@@ -79,3 +79,5 @@ trackbacks:
 - [Simplified Chinese translation](https://sd.csdn.net/a/20120223/312315.html)
 - [Traditional Chinese translation](http://www.hellogcc.org/?p=20)
 - [Linuxfr.org blog post (French)](https://linuxfr.org/news/version-1-0-de-julia)
+
+[^1]: This was originally written as being "time for a 1.0 release." Version 1.0 wasn't released until [much later](https://julialang.org/blog/2018/08/one-point-zero/).
```

Comment:
I wanted to say something snarky like:

> This was originally written as being "time for a 1.0 release," but that turned out to be overly greedy.

but I decided to keep it to the facts.

Reply:
I kind of like the snarky version. No one minds a little self-deprecating snark :)

Reply:
```suggestion
[^1]: This was originally written as being "time for a 1.0 release," but that turned out to be overly greedy. Version 1.0 wasn't released until [much later](https://julialang.org/blog/2018/08/one-point-zero/).
```

Reply:
```suggestion
[^1]: This was originally written as being "time for a 1.0 release" but it turns out this was a bit too greedy: Julia 1.0 wasn't released until [considerably later](https://julialang.org/blog/2018/08/one-point-zero/).
```